modules/ROOT/nav.adoc

```diff
@@ -1,60 +1,99 @@
-* xref:limitations-self.adoc[Requirements and Limitations]
-* xref:deploy-resource-allocation-self-managed.adoc[Resource Allocation]
-* xref:index-vm-bare-metal.adoc[Preparing your VM/Bare Metal Cluster for Runtime Fabric Installation]
-* xref:secure-runtime-fabric.adoc[Secure Runtime Fabric]
-** xref:authorized-namespaces.adoc[Configure Authorized Namespaces]
-** xref:enable-fips-140-2-compliance.adoc[Enable FIPS 140-2 Compliance Mode]
-** xref:kubernetes-hardening-guide.adoc[Kubernetes Hardening Guide]
-** xref:security-architecture.adoc[MuleSoft Anypoint Runtime Fabric Security Architecture]
-** xref:rtfctl-permissions.adoc[]
-** xref:configure-local-registry.adoc[Use a Local Registry]
-* xref:install-self-managed-network-configuration.adoc[Network Configuration]
-** xref:manage-proxy-self.adoc[Manage Proxies]
-* xref:install-index.adoc[Install Runtime Fabric]
-  ** xref:install-self-managed.adoc[Install Runtime Fabric Using rtfctl]
-  ** xref:install-helm.adoc[Install Runtime Fabric Using Helm]
-  ** xref:install-openshift.adoc[Install Runtime Fabric as a Kubernetes (K8s) Operator]
-  ** xref:install-multiple-instances.adoc[Install Multiple Instances on a Single Cluster]
-  ** xref:install-multiple-instances-openshift.adoc[Install Multiple Instances on a Single OpenShift Cluster]
-* xref:upgrade-index.adoc[Upgrade Runtime Fabric]
-  ** xref:upgrade-self-managed.adoc[Upgrade Runtime Fabric Using rtfctl]
-  ** xref:upgrade-helm.adoc[Upgrade Runtime Fabric Using Helm]
-  ** xref:upgrade-openshift.adoc[Upgrade Runtime Fabric as a Kubernetes (K8s) Operator]
-* xref:manage-index.adoc[Manage Runtime Fabric]
-  ** xref:view-health.adoc[View Runtime Fabric Health Details]
-  ** xref:install-rtfctl.adoc[Use the Runtime Fabric Command Line Tool]
-  ** xref:replace-license-key.adoc[Replace a Mule License Key]
-  ** xref:manage-backup-restore.adoc[Back Up and Restore Runtime Fabric]
-  ** xref:enable-fips-140-2-compliance.adoc[Enable FIPS 140-2 Compliance Mode]
+* xref:setting-up-runtime-fabric.adoc[Setting Up Runtime Fabric]
+ ** xref:index-vm-bare-metal.adoc[Preparing your VM/Bare Metal Cluster for Runtime Fabric Installation]
+ ** xref:install-self-managed-network-configuration.adoc[Configuring Network]
+ ** xref:install-index.adoc[Installing Runtime Fabric]
+    *** xref:install-self-managed.adoc[Installing Runtime Fabric Using RTFCTL]
+    *** xref:install-helm.adoc[Installing Runtime Fabric Using Helm]
+    *** xref:install-openshift.adoc[Installing Runtime Fabric as a Kubernetes (K8s) Operator]
+    *** xref:install-multiple-instances.adoc[Installing Multiple Instances on a Single Cluster]
+    *** xref:install-multiple-instances-openshift.adoc[Installing Multiple Instances on a Single OpenShift Cluster]
+ ** xref:configuring-runtime-fabric.adoc[Configuring Runtime Fabric]
+  *** xref:associate-environments.adoc[Associating Business Groups and Environments]
+  *** xref:install-rtfctl.adoc#configure-rtfctl[Configuring Runtime Fabric Using RTFCTL]
```

Comment:
It seems you are adding an anchor as an entry in the nav.adoc, is this intended? I never tried it, IDK if this works. In addition, this same entry repeats in line 75, there seems to be some duplication in this nav.adoc. Please check.

Reply:
Yes, it is intended. As it is pointing to some sections in the RTFCTL Reference. I've tried it and it works fine.
Also, some links repeat in the last section of the nav that belongs to Using RTFCTL Tool. We'll make a test to see if users prefers a mini product-feature TOC.